modules/nw-cudn-localnet.adoc

```diff
@@ -0,0 +1,79 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-localnet_{context}"]
+= Creating a ClusterUserDefinedNetwork CR for Localnet
+
+A `Localnet` topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+.Prerequisites
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have created and configured the Open vSwitch (OVS) bridge mapping to associate the logical OVN-Kubernetes network with the physical node network through the OVS bridge. For more information, see "Configuration for a localnet switched topology".
+
+.Procedure
+
+. Create a request for `Localnet` topology type cluster-wide user-defined network:
+
+.. Create a YAML file, such as `cluster-udn-localnet.yaml`, to define your request for a `localnet` topology as in the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: <cudn_name> # <1>
+spec:
+  namespaceSelector: # <2>
+    matchLabels: # <3>
+      "<label_1_key>": "<label_1_value>" # <4>
+      "<label_2_key>": "<label_2_value>" # <4>
+  network: # <5>
+    topology: Localnet # <6>
+    localnet: # <7>
+      role: Secondary # <8>
+      physicalNetworkName: test
+      ipam: {lifecycle: Persistent}
+      subnets: ["192.168.0.0/16", "2001:dbb::/64"] # <9>
+----
+<1> Name of your `ClusterUserDefinedNetwork` CR.
+<2> A label query over the set of namespaces that the cluster UDN CR applies to. Uses the standard Kubernetes `MatchLabel` selector. Must not point to `default` or `openshift-*` namespaces.
```

Comment:
I'd say it must not point to `default`, `openshift-*` or any other system namespaces. Reason: there are some namespaces, like `kube-system` or `metallb` that don't match this pattern and also mustn't be touched by this, yet I don't think it is worth it to maintain an exhaustive list of all of them.